### ISSUE TITLE
Custom objective for multi-output regression with multi_strategy=multi_output_tree

### ISSUE THREAD

**Issue by sergun** [dmlc/xgboost#10060]
How does custom objective work for multi-output regression with multi_strategy=multi_output_tree?

Custom loss function is calculated on each output separately and then summed up?

Or is it called once, so it is possible e.g. to define loss function as weighted sum of losses applied to each output?

**Comment by trivialfis** [comment]
Hi, the `multi_output_tree` is working in progress, we don't recommend using a custom objective with it since we haven't done extensive tests yet.

**Comment by trivialfis** [comment]
Closing in favor of https://github.com/dmlc/xgboost/issues/9043

**Comment by RawabiAlwanin** [comment]
how gradients are combined in multi-target models using built in objectives like reg:squarederror? are they summed up?
In which file can I find this peace of information?

**Comment by RawabiAlwanin** [comment]
how gradients are combined in multi-target models using built in objectives like reg:squarederror? are they summed up? and
in which file can find this gradients combination?
